dask/dataframe/io/tests/test_io.py

```diff
@@ -509,9 +509,14 @@ def test_to_bag():
         index=pd.Index([1.0, 2.0, 3.0, 4.0], name="ind"),
     )
     ddf = dd.from_pandas(a, 2)
+    tuple_to_dict = lambda x: dict(x._asdict())
```

Comment:
I would rather that we test against the final output form, rather than a copy of our code.  So I would expect this to look like ...

```python
expected = [
    {"...": ..., ...},
    {"...": ..., ...},
    {"...": ..., ...},
]

assert ddf.to_bag.compute() == expected
```

Reply:
make sense, got it :)